webpage/templates/source_detail.txt

```diff
@@ -6,13 +6,21 @@ Source: {{ source.common_name }}
 Basic source info
 -----------------
 
-TODO
+{% if source.source_id < 10 %} 
+`Information <https://github.com/gammapy/gamma-cat/blob/master/input/sources/tev-00000{{ source.source_id }}.yaml>`__
+{% endif %}
+{% if source.source_id > 9 and source.source_id < 100 %}
+`Information <https://github.com/gammapy/gamma-cat/blob/master/input/sources/tev-0000{{ source.source_id }}.yaml>`__
+{% endif %}
+{% if source.source_id > 100 %}
+`Information <https://github.com/gammapy/gamma-cat/blob/master/input/sources/tev-000{{ source.source_id }}.yaml>`__
+{% endif %}
 
 SEDs
 ----
 
 {% for resource in resources.resources if resource.type == 'sed' %}
-* `{{ resource.reference_id }} <../../output/{{ resource.url }}>`__
+* {{ resource.reference_id }} (`Show <https://github.com/gammapy/gamma-cat/tree/master/output/{{ resource.url }}>`__, `Download <../../output/{{ resource.url }}>`__, `Gamma-Cat Input <https://github.com/gammapy/gamma-cat/tree/master/input/data/{{ resource.year }}/{{ resource.parsedref }}/tev-{{ resource.source_str }}-sed.ecsv>`__)
```

Comment:
@cdeil Do you have better names for "Download", "Gamma-Cat Input" and "Show"?

Reply:
Suggest "Github input" and "Github output"